app/controllers/cafe_controller.rb

```diff
@@ -0,0 +1,30 @@
+# encoding:UTF-8
+class CafeController < ApplicationController
+  skip_authorization
+
+  def index
+  end
+
+  def competition
+    lp = params[:lp] || '1'
+    year = Time.zone.now
+    if params[:year].present?
+      year = Time.zone.local(params[:year])
+    end
+
+    @competition = CafeCompetition.new(CafeQueries.cafe_workers(lp, year),
+                                       CafeQueries.working_users(lp, year),
+                                       lp, year)
+  end
+
+  def ladybug
+    authorize!(:ladybug, CafeShift)
+    if ladybug = params[:ladybug].present? && params[:ladybug][:date].present?
```

Comment:
Useless assignment to variable - `ladybug`.<br>Use the return of the conditional for variable assignment and comparison.